### ISSUE TITLE
R_air is named incorrectly

### ISSUE THREAD

**Issue by jensdebruijn** [Deltares/hydromt#1044]
### HydroMT version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://github.com/Deltares/hydromt)

### Kind of issue

Docs are wrong

### Location of the documentation

https://github.com/Deltares/hydromt/blob/a707d44497ef493174b76fafde90e45085d78a6c/hydromt/workflows/forcing.py#L480

### Documentation problem

I believe this should be named the universal gas constant rather than the specific gas constant for dry air. The calculation itself is correct with the given values.

**Comment by PeterNelemans** [comment]
Agree; this is just the universal gas constant. The variable should therefore also be renamed to just `R` imho. Dividing the universal gas constant `R` by the molucalar mass of dry air `Mo` will give you the specific gas constant for dry air (287 J kg⁻¹ K⁻¹), which you could denote with `R_air`, hence I guess the confusion.
